Fix m4 quoting to avoid an infinite loop in m4 with recent autoconf/automake

RP 1/2/10

Index: libsndfile-1.0.17/acinclude.m4
===================================================================
--- libsndfile-1.0.17.orig/acinclude.m4	2010-02-01 14:53:35.517826746 +0000
+++ libsndfile-1.0.17/acinclude.m4	2010-02-01 14:53:51.409122706 +0000
@@ -255,7 +255,7 @@
 
 )
 ]
-
+[
 if test $ac_cv_c_byte_order = big ; then
 	ac_cv_c_big_endian=1
 	ac_cv_c_little_endian=0
@@ -272,7 +272,7 @@
 	AC_MSG_WARN([[*** src/config.h may need to be hand editied.                    ]])
 	AC_MSG_WARN([[*****************************************************************]])
 	fi
-
+]
 )# AC_C_FIND_ENDIAN
 
 
